java-agent/src/nativeMain/kotlin/com/epam/drill/agent/configuration/process/JavaProcess.kt

```diff
@@ -24,5 +24,11 @@ data class JavaProcess(
     var javaParams: List<String>? = null
 ) {
     val firstAgentPath
-        get() = nativeAgents.first().split("=").first().replace('/', '\\').substringBeforeLast('\\')
+        get() = nativeAgents
+            .find { it.contains("drill_agent") }

```

Comment:
Can we declare this as a value in `gradle.properties` and then use both here and in the build script?
Just to ensure this code won't break if artifact name will change.
I think it will be hard to debug and will cost us an hour or two :)

Reply:
This is a good idea. There is even the property `nativeAgentLibName`. But the implementation will not be easy due to Kotlin Native. The easiest way to get access to `gradle.properties` this is to repeat the trick with the `agentVersion` property from generated `Version.kt` file. But before that I would like to discuss the details.